alpaca_trade_api/stream2.py

```diff
@@ -370,6 +378,19 @@ async def close(self, renew):
                                            self._oauth,
                                            raw_data=self._raw_data)
 
+    async def signal_stop_ws(self):
+        """
+        Signal the ws connections to stop.
+        We set a signal in a Queue and then call cancel_task
```

Comment:
[minor] this is an implementation detail that I wouldn't include in the docstring.

Reply:
done. removed internal implementation from doc string